### ISSUE TITLE
mission_raw_import_and_fly.cpp inconsistency in frame type 

### ISSUE THREAD

**Issue by alariq** [mavlink/MAVSDK#1915]
`create_mission_item` has `_frame` parameter which is set to 3 ([MAV_FRAME_GLOBAL_RELATIVE_ALT](https://mavlink.io/en/messages/common.html#MAV_FRAME_GLOBAL_RELATIVE_ALT)) though `lat` and `lon` are converted as if they are prepared for 6 ([MAV_FRAME_GLOBAL_RELATIVE_ALT_INT](https://mavlink.io/en/messages/common.html#MAV_FRAME_GLOBAL_RELATIVE_ALT_INT)).

I understand that x,y,z params are integer anyway and it looks like they should be multiplied by `1e7` (i.e. converted to int) in both cases, but then what is the difference between those 2 frame types?
Should description in .xml file for `MAV_FRAME_GLOBAL_RELATIVE_ALT` be updated to also mention convertion to integer?
Should integration test change frame type from 3 to 6 (or even better use a symbolic constant)? or at least change comment on line [191](https://github.com/mavlink/MAVSDK/blob/7ebcc681eaf425c5426d12bc53b529bb90062f5c/src/integration_tests/mission_raw_import_and_fly.cpp#L191)

Thank you!

**Comment by julianoes** [comment]
Good point! This looks wrong but it's probably working correctly because the MAVLink message type itself is `_INT` and so the autopilot knows how to interpret the data.

I'm not sure if this was on purpose given what was expected by PX4, or done by QGC, and so it just matches it, or if this is an oversight and wrong (but working).

It would be interesting if you could check if this matters and what QGC does.

**Comment by PavelGuzenfeld** [comment]
The offending file (mission_raw_import_and_fly.cpp) has been removed. Current code in mission_changed.cpp correctly uses frame 6 (MAV_FRAME_GLOBAL_RELATIVE_ALT_INT). Can this be closed?

**Comment by julianoes** [comment]
There is also this test: https://github.com/mavlink/MAVSDK/blob/main/cpp/src/mavsdk/plugins/mission_raw/mission_import_test.cpp

Let's close and re-open in case it comes up again.
